cmake/plan.sh

```diff
@@ -11,23 +11,33 @@ pkg_shasum=80d0faad4ab56de07aa21a7fc692c88c4ce6156d42b0579c6962004a70a3218b
 pkg_deps=(
   core/glibc
   core/gcc-libs
+  core/curl
+  core/zlib
 )
 pkg_build_deps=(
   core/coreutils
   core/diffutils
   core/make
   core/gcc
-  core/curl
-  core/zlib
-  core/bzip2
 )
 
 pkg_lib_dirs=(lib)
 pkg_include_dirs=(include)
 pkg_bin_dirs=(bin)
 
 do_build() {
-  ./bootstrap
+  ZLIB=$(pkg_path_for core/zlib)
+  ZLIB_LIB="${ZLIB}/lib"
+  ZLIB_INCLUDE="${ZLIB}/include"
+  CURL=$(pkg_path_for core/curl)
+  CURL_LIB="${CURL}/lib"
+  CURL_INCLUDE="${CURL}/include"
+
+  LD_LIBRARY_PATH=${ZLIB_LIB}:${CURL_LIB}
+  ./bootstrap --parallel="$(nproc)" --system-curl -- -DLD_LIBRARY_PATH="${LD_LIBRARY_PATH}" \
```

Comment:
I don't think LD_LIBRARY_PATH is needed here, since you are explicitly setting the CURL and ZLIB paths below.   It's a runtime setting (In this case the runtime of the build process) and is not going to guarantee that the built package is going to be able to find appropriate libraries later.     

Additionally, it could introduce some odd behavior into the build process, as described [in this post in the forums](https://forums.habitat.sh/t/cannot-attach-in-a-plan-where-cflags-library-path-or-ld-library-path-are-modified/650/3)

Reply:
.Updated in ebe721e